common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -77,37 +84,48 @@
             }
         ],
         "@typescript-eslint/consistent-type-definitions": [
-            "error"
+            "error",
+            "interface"
+        ],
+        "@typescript-eslint/consistent-type-imports": [
+            "warn",
+            {
+                "prefer": "type-imports"
+            }
         ],
         "@typescript-eslint/dot-notation": [
             "error"
         ],
-        "@typescript-eslint/explicit-function-return-type": [
+        "@typescript-eslint/eol-last": [
             "off"
         ],
+        "@typescript-eslint/explicit-function-return-type": [
+            "warn",
+            {
+                "allowConciseArrowFunctionExpressionsStartingWithVoid": false,
+                "allowDirectConstAssertionInArrowFunctions": true,
+                "allowExpressions": true,
+                "allowHigherOrderFunctions": true,
+                "allowTypedFunctionExpressions": true
+            }
+        ],
         "@typescript-eslint/explicit-member-accessibility": [
-            "off"
+            "error",
+            {
+                "accessibility": "explicit"
+            }
         ],
         "@typescript-eslint/explicit-module-boundary-types": [
-            "off"
-        ],
-        "@typescript-eslint/func-call-spacing": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/indent": [
             "off"
         ],
-        "@typescript-eslint/interface-name-prefix": [
-            "off"
-        ],
-        "@typescript-eslint/keyword-spacing": [
+        "@typescript-eslint/linebreak-style": [
             "off"
         ],
         "@typescript-eslint/member-delimiter-style": [
```

Comment:
I thought member-delimiter-style was already on?  Or is it enforced by some semicolon rule?  In any case, good one to error I think.

Reply:
I have a perhaps unjustified dislike of formatting rules in lint. I'll prioritize getting tsfmt running. It's already mostly done (try `npm run tsfmt:fix` from the repo root).

I filed #9756 to track.